runtime/storage-proxy.js

```diff
@@ -157,8 +159,24 @@ class StorageProxyBase {
   }
 
   _processUpdates() {
+
+    let updateIsNext = update => {
+      if (update.version == this._version + 1) {
+        return true;
+      }
+      // Holy Layering Violation Batman
```

Comment:
Add a brief comment on whether this is expected to be here long term or if there's other work planned to improve?